runtime/precompiles/src/lib.rs

```diff
@@ -115,7 +115,7 @@ where
 	<R::Call as Dispatchable>::Origin: From<Option<R::AccountId>>,
 	// Next four are copied from staking.rs. I guess they need to be duplicated here...
 	R: parachain_staking::Config + pallet_evm::Config,
-	R::AccountId: From<H160>, //TODO Can't we somewhere specify R<AccountId = H160> instead of this bound?
+	R::AccountId: From<H160>,//TODO Can't we specify R<AccountId = H160> instead of this bound?
```

Comment:
To reply to this comment which I just truncated, we can't specify the exact assignment because it's a trait bound, not a trait assignment